pkg/webui/console/components/webhook-form/index.js

```diff
@@ -184,16 +195,52 @@ export default class WebhookForm extends Component {
     this.setState({ displayOverwriteModal: false })
   }
 
+  @bind
+  async handleReactivate() {
+    const { onReactivate, onReactivateSuccess } = this.props
+    const healthStatus = {
+      health_status: null,
+    }
+
+    try {
+      await onReactivate(healthStatus)
+      onReactivateSuccess()
+    } catch (error) {
+      this.setState({ error })
+    }
+  }
+
   render() {
-    const { update, initialWebhookValue, webhookTemplate } = this.props
+    const {
+      update,
+      initialWebhookValue,
+      webhookTemplate,
+      suspendedWebhookMessage,
+      reactivateButtonMessage,
+    } = this.props
     const { error, displayOverwriteModal, existingId } = this.state
     let initialValues = blankValues
     if (update && initialWebhookValue) {
       initialValues = mapWebhookToFormValues(initialWebhookValue)
     }
 
+    const mayReactivate =
+      update &&
+      initialWebhookValue &&
+      initialWebhookValue.health_status &&
+      initialWebhookValue.health_status.unhealthy
+
     return (
       <>
+        {mayReactivate && (
+          <Notification
+            warning
+            content={suspendedWebhookMessage}
+            action={this.handleReactivate}
+            actionMessage={reactivateButtonMessage}
+            buttonIcon="refresh"
+          />
```

Comment:
```suggestion
            small
          />
```

We typically use the small style.